microservices/netinfra_utils/importDevices.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+from collections import namedtuple
+from sys import argv
+import subprocess
+import json
+
+
+def main():
```

Comment:
pls, add short description about purpose of the script.

Reply:
This script replaces the job of 'newman'. It posts data into elasticsearch so that the devices with their data from the csv files are available in the kibana inventory.